stdlib/sys.pyi

```diff
@@ -27,12 +27,11 @@ _T = TypeVar("_T")
 # The following type alias are stub-only and do not exist during runtime
 _ExcInfo = Tuple[Type[BaseException], BaseException, TracebackType]
 _OptExcInfo = Union[_ExcInfo, Tuple[None, None, None]]
-_PathSequence = Sequence[Union[bytes, str]]
+_ModulePath = Sequence[str]
```

Comment:
Nit: This type alias is no longer needed, because it is used only once. Just use `Sequence[str]` directly.